src/Hooks/useFetchStats.js

```diff
@@ -37,11 +37,22 @@ export function useTrackStats(timeBounds) {
 			orderBy('unix_time', 'asc')
 		);
 		const unsubscribe = onSnapshot(q_states, (snapshot) => { // real-time listener for changes to the queried stats data
-			setStats(convertStatsSnapshot(snapshot));
+			setStats(convertStatsSnapshot(snapshot)); //this is the initial data fetch
 			setLoading(false);
+      console.log("hi")
 		});
-		// make sure to unsubscribe when the component unmounts cause idek but like efficiency probably
-		return unsubscribe;
+    // Creates an interval which will update the current data every minute
+    const timescale = timeBounds[1] - timeBounds[0]
```

Comment:
I know we discussed that the timebounds are used elsewhere (AreaGraph), but it still feels awkward to pass in the two time bounds when the only thing we need is the timescale, which we already have, up in Dashboard. What if we just passed `timescale` down all the way through all components, and only calculate what we need? That would be once in this file to see at what time to start the query, and once in AreaGraph to get the min/max for the domain.